Interfaces/scripts/dirac-dms-get-files.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+########################################################################
+# $HeadURL$
+# File :    dirac-dms-get-file
+# Author :  Stuart Paterson
+########################################################################
```

Comment:
Indeed ;-) This was copied across when I duplicated the dirac-dms-get-file.py. I'll change this.